server/graphql/v1/mutations/collectives.js

```diff
@@ -177,6 +178,74 @@ export async function createCollective(_, args, req) {
   return collective;
 }
 
+export async function createCollectiveFromGithub(_, args, req) {
+  if (!req.remoteUser) {
+    throw new errors.Unauthorized({
+      message: 'You need to be logged in to create a collective',
+    });
+  }
+
+  if (!args.collective.name) {
+    throw new errors.ValidationFailed({ message: 'collective.name required' });
+  }
+
+  let collective;
+  const collectiveData = { ...args.collective };
+  const user = await models.User.findByPk(req.remoteUser.id);
```

Comment:
Isn't `req.remoteUser` already an user?

Reply:
Yeah it is the same thing, just trying too be a little defensive.

Reply:
We don't have to here. We should be confident in `req.remoteUser`.